pkg/operator/ceph/object/user.go

```diff
@@ -227,3 +235,42 @@ func UnlinkUser(c *Context, id, bucket string) (string, int, error) {
 	}
 	return result, RGWErrorNone, nil
 }
+
+// EnableUserQuota will allows to enable quota defined for a user
+func EnableUserQuota(c *Context, id string) (string, error) {
+	logger.Debug("Enabling user quota for %q", id)
+	args := append([]string{"quota", "enable", "--quota-scope", "user", "--uid", id})
+	result, err := runAdminCommand(c, args...)
+	if err != nil {
+		if strings.Contains(result, "uid is required for quota operation") {
```

Comment:
same comment as Travis made regarding hard-coding error message fragments to make logic decisions. It maybe be that ceph is not providing sufficiently granular error codes? If that's the case perhaps an issue against ceph error handling should be created? @travisn ?

Reply:
okay, will address in next PR